nav2_map_server/CMakeLists.txt

```diff
@@ -14,130 +14,189 @@ find_package(std_msgs REQUIRED)
 find_package(tf2 REQUIRED)
 find_package(nav2_util REQUIRED)
 find_package(GRAPHICSMAGICKCPP REQUIRED)
+find_package(sensor_msgs REQUIRED)
+find_package(pcl_conversions REQUIRED)
+find_package(Eigen3 3.3 REQUIRED NO_MODULE)
```

Comment:
Why NO_MODULE?

Reply:
https://eigen.tuxfamily.org/dox/TopicCMakeGuide.html

Reply:
NO_MODULE means skip FindEigen3.cmake (the find module), go straight for Eigen3Config.cmake or eigen3-config.cmake (the config file) as eigen3 is now providing native cmake support.